docs/using/ruby.rst

```diff
@@ -32,7 +32,7 @@ let's call it "blog":
 
 ::
 
-    $ tsuru app create blog ruby
+    $ tsuru app create blog ruby --team
```

Comment:
this is not accurate, if the user runs this exact command tsuru will ask the user to specify a valid team